### ISSUE TITLE
Sección de cómo descargar datos programáticamente

### ISSUE THREAD

**Issue by eliocamp** [dnme-minturdep/DT6_ciencia_de_datos_turismo#8]
Si vamos a usar la base de datos del ministerio, les parece agregar una sección de cómo descargar esos datos desde R? Por lo que ví son todos csv, así que en realidad no hay mucho para hacer. Poner `read_csv(url)` y agregar que es mejor hacer un `download.file()` condicional a que no exista el archivo para no depende de tener internet y para no estar descargando el mismo archivo una y otra vez.

**Comment by eliocamp** [comment]
Exacto, pero eso sería la intención para de alguna manera "fijar" los datos y que no cambien constantemente y que el análisis sea un poco más reproducibel.

**Comment by yabellini** [comment]
Esto está relacionado con el issue #1
